src/modules/map/components/olMap/handler/highlight/styleUtils.js

```diff
@@ -74,4 +75,86 @@ export const highlightTemporaryGeometryFeatureStyleFunction = () => {
 	return [hlStyle];
 };
 
+export const highlightAnimatedCoordinateFeatureStyleFunction = () => {
+
+	const selectStroke = new Stroke({
+		color: [50, 128, 0, 1],
+		width: 3
+	});
+
+	const selectFill = new Fill({
+		color: [50, 255, 0, 0.3]
+	});
+
+	const selectStyle = new Style({
+		fill: selectFill,
+		stroke: selectStroke,
+		image: new CircleStyle({
+			radius: 10,
+			fill: selectFill,
+			stroke: selectStroke
+		})
+	});
+
+	return [selectStyle];
+};
+
+export const createAnimation = (map, feature, endCallback) => {
+	const duration = 1500; // 1 second
+	const start = Date.now();
+
+	const animate = (event) => {
+		const vectorContext = getVectorContext(event);
+		const frameState = event.frameState;
+		const flashGeom = feature.getGeometry().clone();
+		const elapsed = frameState.time - start;
+		// don't allow negative values for radius
+		const elapsedRatio = (elapsed >= 0 ? elapsed : 0) / duration;
+		// radius will be 6 at start and 30 at end.
+		const radius = easeOut(elapsedRatio) * 24 + 6;
+		const opacity = easeOut(1 - elapsedRatio);
+
+		const getStyles = (radius, opacity, index) => {
+			const style = new Style({
+				image: new CircleStyle({
+					radius: radius,
+					stroke: new Stroke({
+						color: index === 0 ? 'rgba(3, 3, 3, 0.2)' : 'rgba(255, 255, 255, ' + opacity + ')',
+						width: 0.4 + opacity
+					}),
+					fill: new Fill({
+						color: 'rgba(9, 157, 221, ' + opacity + ')'
+					})
+				})
+			});
+
+
+			return radius >= 6 + 8 ? [style, ...getStyles(radius - 8, opacity, index + 1)] : [style];
+		};
+
+		getStyles(radius, opacity, 0).forEach(style => {
+			vectorContext.setStyle(style);
+			vectorContext.drawGeometry(flashGeom);
+		});
+
+
+		if (elapsed > duration) {
+			endCallback();
+			return;
+			// TODO: a alternative solution, when the caller wants to implement a permanent
+			// blinking until a defined end is reached, so the endCallback must return true/false
+			// if (endCallback()) {
+			// 	return;
+			// }
+			// else {
+			// 	start = Date.now();
+			// }
+
```

Comment:
@thiloSchlemmer
 I think we can remove that part. If a caller wants to stop the animation, he removes the corresponding animated HighlightFeature and optionally replaces it with a permanent HighlightFeature.
In contrast, the animation should be running endlessly until that point.